From 6fca0677d22675401c332cf71fb3989ccd0071c8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcin=20=C5=9Alusarz?= <marcin.slusarz@intel.com>
Date: Thu, 23 Feb 2017 17:53:35 +0100
Subject: [PATCH] common: let user build debs with Valgrind enabled

---
 utils/build-dpkg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/build-dpkg.sh b/utils/build-dpkg.sh
index 825029dae..cb29145f7 100755
--- a/utils/build-dpkg.sh
+++ b/utils/build-dpkg.sh
@@ -690,7 +690,7 @@ tail -n1 $CHANGELOG_TMP >> debian/changelog
 rm $CHANGELOG_TMP
 
 # This is our first release but we do
-debuild -us -uc
+debuild --preserve-envvar=EXTRA_CFLAGS --preserve-envvar=EXTRA_LDFLAGS -us -uc
 
 cd $OLD_DIR
 
-- 
2.11.0

